showcase/plot/scatterplot.js

```diff
@@ -42,6 +42,7 @@ export default class Example extends React.Component {
         <MarkSeries
           className="mark-series-example"
           strokeWidth={2}
+          opacity="0.8"
```

Comment:
I guess 0.8 (number) makes sense from a functional point of view. Then again, opacity is stored in the CSSStyleDeclaration object as a string. So imo neither or them is vastly superior to the other.